tests/integration/iop-ambient-test-defaults.yaml

```diff
@@ -10,22 +10,12 @@ spec:
       - name: istio-ingressgateway
         enabled: true
   values:
-    cni:
-      ambient:
-        # Some of the tests require DNS capture
-        # For that, DNS capture must be enabled in the CNI
-        # and DNS proxying must be enabled in ztunnel
-        dnsCapture: true
-    ztunnel:
-      meshConfig:
-        defaultConfig:
-          proxyMetadata:
-            ISTIO_META_DNS_CAPTURE: "true"
     meshConfig:
-      defaultConfig:
-        proxyMetadata:
-          ISTIO_META_DNS_CAPTURE: "true"
-          DNS_PROXY_ADDR: "0.0.0.0:15053"
+      # maybe necessary for Sidecar interoperability testing
```

Comment:
we should remove it instead of comment.

Reply:
thanks, I wanted to see CI run before I totally removed it. I'll get rid of it now.